docs/SUMMARY.md

```diff
@@ -9,6 +9,7 @@
   * [Install Devtron on Minikube, Microk8s, K3s, Kind](setup/install/Install-devtron-on-Minikube-Microk8s-K3s-Kind.md)
   * [FAQ on Installation](setup/install/faq-on-installation.md)
   * [Uninstall Devtron](setup/install/uninstall-devtron.md)
+* [Devtron Kubernetes Client](setup/install/install-devtron-Kubernetest-client.md)
```

Comment:
Kubernetes spelling is wrong in the link and reference, please correct.

Reply:
corrected.